vaadinfrontend/src/main/java/life/qbic/datamanager/views/PersonElement.java

```diff
@@ -0,0 +1,50 @@
+package life.qbic.datamanager.views;
+
+import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import life.qbic.projectmanagement.domain.project.PersonReference;
+
+import java.util.Objects;
+
+/**
+ * <b>short description</b>
+ *
+ * <p>detailed description</p>
+ *
+ * @since <version tag>
+ */
+public class PersonElement extends VerticalLayout {
```

Comment:
I could also include the dropbox here in this element and we could swap it in the element itself, what do you think?